.github/automation/commit-msg-check.py

```diff
@@ -20,29 +20,17 @@
 
 import argparse
 import subprocess
+import re
 
 # * Ensuring the scopes end in colon and same level scopes are comma delimited.
 # TODO: Limit scopes to an acceptable list of tags.
```

Comment:
Please add to this comment that the current implementation can only validate the top level scope due to ambiguous parsing of the `:` token between the scope title and the description.